ocdskingfisherarchive/database_archive.py

```diff
@@ -0,0 +1,33 @@
+import sqlite3
+
+
+class DataBaseArchive:
+
+    def __init__(self, config):
+        self.config = config
+        self.conn = sqlite3.connect(config.database_archive_filepath)
+
+        c = self.conn.cursor()
+        c.execute("SELECT name FROM sqlite_master WHERE type='table' AND name='collections'")
+        if not c.fetchone():
+            c.execute("CREATE TABLE collections (collection_id INTEGER PRIMARY KEY NOT NULL, state TEXT NOT NULL)")
+            c.close()
+            self.conn.commit()
+        else:
+            c.close()
```

Comment:
Prefer the pattern in https://github.com/open-contracting/kingfisher-views/blob/master/ocdskingfisherviews/db.py where we reuse a global connection and cursor instead of carefully getting and closing a new one each time.